security/dirtycow.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "$(dirname $0)/lib.sh"
+
+container_name="dirtycow"
+runtime=${RUNTIME:-kata-runtime}
+test_repo="https://github.com/dirtycow/dirtycow.github.io"
+test_dir="/root/dirtycow"
+test_file="$test_dir/test"
+test_file_content="Hello"
+dirty_file_content="pwned"
+
+finish() {
+	docker rm -f "$container_name"
+}
+trap finish EXIT
+
+# Run a gcc container
+docker run --runtime="$runtime" --name="$container_name" -dti gcc bash
+
+# Turning off periodic writeback makes exploit stable
+# This should fail inside a container
+docker exec "$container_name" bash -c "echo 0 > /proc/sys/vm/dirty_writeback_centisecs"
+[ $? == 0 ] && die "Turned off periodic writeback"
+
+# Clone dirtycow repo
+docker exec "$container_name" git clone "$test_repo" "$test_dir"
```

Comment:
It seems unlikely that this will be changed again but if it does, we might want to consider adding a particular `HEAD` commit to `versions.yaml`.